Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:lts-slim
+
+RUN mkdir -p /app/test 
+
+WORKDIR /app
+
+ENV NPM_CONFIG_PREFIX=/home/node/.npm-global \
+    PNPM_HOME=/.pnpm \ 
+    PATH=${PATH}:${PNPM_HOME}:/home/node/.npm-global/bin
+
```

Comment:
Siempre conviene que pongáis el comando lo más cerca posible de donde se afecta. `WORKDIR` no afecta `ENV`, por lo que no tiene mucho sentido ponerlo justo encima.